src/SSymmetricCompact.jl

```diff
@@ -0,0 +1,239 @@
+struct SSymmetricCompact{N, T, L} <: StaticMatrix{N, N, T}
```

Comment:
Needs docs about the storage order or `lowertriangle`.

Reply:
Added docs.